Framework/src/Bookkeeping.cxx

```diff
@@ -0,0 +1,61 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   Bookkeeping.cxx
+/// \author Barthelemy von Haller
+///
+
+#include "QualityControl/Bookkeeping.h"
+#include "QualityControl/QcInfoLogger.h"
+#include "BookkeepingApi/BkpProtoClientFactory.h"
+
+using namespace o2::bkp::api::proto;
+
+namespace o2::quality_control::core
+{
+
+void Bookkeeping::init(std::string url)
```

Comment:
Once DPL is able to run several Data Processors multithreaded, we will initialize the BK several times for no reason. Perhaps you can add a check if Bookkeeping was already initialized with the same url and do nothing then, plus if the url is different than before, add a warning, but proceed?

Reply:
yes, done